### PR TITLE
Adding permission of author acknowledgement to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by CiaraFlanagan**
Added the heading and sentence:

## Acknowledgement of Contributions

If you list yourself as the author of a pattern, we assume we have your permission to mention you when referring to the respective pattern.

### CODE REVIEW

**Comment by CiaraFlanagan** [comment]
I am not able to assign anyone to review and merge this pull request.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -35,6 +35,11 @@ Join the InnerSource Commons [Slack instance](https://isc-inviter.herokuapp.com/
 
 The contents of this repository are licensed under [CC-BY-SA-4.0](LICENSE.txt). By contributing to this repository, you grant us (and everyone else for that matter) the right to use your contribution in accordance with that license.
 
+## Acknowledgement of Contributions
+
+If you list yourself as the author of a pattern, we assume we have your permission to mention you when referring to the respective pattern.
+
+
```
Comment (spier) [inline CONTRIBUTING.md]:
```suggestion
```

**Review by spier — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -35,6 +35,10 @@ Join the InnerSource Commons [Slack instance](https://isc-inviter.herokuapp.com/
 
 The contents of this repository are licensed under [CC-BY-SA-4.0](LICENSE.txt). By contributing to this repository, you grant us (and everyone else for that matter) the right to use your contribution in accordance with that license.
 
+## Acknowledgement of Contributions
```
Comment (spier) [inline CONTRIBUTING.md]:
```suggestion
## Acknowledgment of Contributions
```

Using American English

**Comment by spier** [comment]
@CiaraFlanagan no need to assign somebody. The Trusted Committers (or rather CODEOWNERS) of the repo are automatically added as reviewers.

Thank you for your contribution!

**Pull request merged**